hdf_compass/compass_viewer/viewer.py

```diff
@@ -236,11 +236,17 @@ def load_plugins():
     except ImportError:
         log.warning("Opendap plugin: NOT loaded")
     
-    from hdf_compass import hdf5rest_model    
+#    from hdf_compass import hdf5rest_model    
```

Comment:
@michaelsippel this looks like it just did not work on your system, but should not be removed in general